proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -280,6 +280,11 @@ message WorkflowExecutionInfo {
     // Should be deprecated once the transition history is fully launched
     repeated VersionedTransition previous_transition_history = 100;
     VersionedTransition last_transition_history_break_point = 101;
+
+    // This is a set of child workflows that were initialized after the reset point in the parent workflow.
```

Comment:
nit: add comment that this includes children in continue as new runs.

Reply:
Added